actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -60,6 +61,7 @@ def initialize(view_context)
         def attributes(attributes)
```

Comment:
```suggestion
        def attributes(attributes) # :doc:
```

Reply:
Thanks @eileencodes.
I can't seem to make that work 😞. (Ruby 3.1, rdoc 6.4.0)
When I search for examples they all seem to document private methods of documented classes, not methods of `:nodoc:`-ed classes.
It seems like a `:nodoc:`-ed class should not be visible in the documentation even if a method is marked as `:doc:`, which sort of makes sense.